src/Microsoft.AspNetCore.Routing/Template/TemplateBinder.cs

```diff
@@ -265,7 +269,7 @@ private bool TryBindValuesCore(UriBuildingContext context, RouteValueDictionary
                             // Example: template = {id}.{format?}. parameters: id=5
                             // In this case after we have generated "5.", we wont find any value 
                             // for format, so we remove '.' and generate 5.
-                            if (!context.Accept(converted, parameterPart.EncodeSlashes))
+                            if (!context.Accept(converted, parameterPart.EncodeSlashes, GetParameterTransformer(parameterPart)))
```

Comment:
Wait OK, I'm confused now, I thought you were saying elsewhere that it made sense for parameter tranformers to only apply staticly?

Reply:
Discussed on the call, makes sense.